samples/nfc/tag_reader/src/main.c

```diff
@@ -37,7 +38,7 @@
 #define NFC_T4T_APDU_MAX_SIZE 1024
 #define TAG_TYPE_4_NLEN_FIELD_SIZE 2
```

Comment:
I think you can remove it and use NDEF file macros once this PR https://github.com/nrfconnect/sdk-nrf/pull/2609 is merged. I guess it would be possible to use just a NDEF File header for it (without including source file via Kconfig).

Reply:
Removed